packages/twenty-server/src/modules/mail-sender/workflow-actions/send-email.workflow-action.ts

```diff
@@ -41,34 +88,34 @@ export class SendEmailWorkflowAction {
         return { result: { success: false } };
       }
 
-      const mainText = Handlebars.compile(step.settings.template)(payload);
+      const body = Handlebars.compile(step.settings.body)(payload);
```

Comment:
I didn't know we were using Handlebars as the templating language for emails (and probably future variables too). That's great!